actions/tag-and-release/action.yaml

```
@@ -0,0 +1,45 @@
+name: "Tag and release"
+description: "Tag and release your repo with target tag."
+
+inputs:
+  tag:
+    description: "The version of the tag to be publish and released e.g 1.0.0"
+    required: true
+  release-title:
+    description: "Text for the github release title"
```

check capitalisation
```suggestion
    description: "Text for the GitHub release title"
```